src/main/java/org/openrewrite/staticanalysis/DefaultComesLastVisitor.java

```diff
@@ -75,7 +75,7 @@ public J.Switch visitSwitch(J.Switch switch_, P p) {
                     aCase.getStatements().isEmpty() && !foundNonEmptyCase) {
                     casesGroupedWithDefault.add(aCase);
                 } else {
-                    if (defaultCase != null && defaultCase.getStatements().isEmpty() && !foundNonEmptyCase) {
+                    if (defaultCase != null && defaultCase.getStatements().isEmpty() && foundNonEmptyCase) {
```

Comment:
```suggestion
                    if (nonNull(defaultCase) && isEmpty(defaultCase.getStatements()) && foundNonEmptyCase) {
```
https://stackoverflow.com/questions/28140193/purpose-of-objects-isnull-objects-nonnull
https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/CollectionUtils.html#isEmpty(java.util.Collection)